lib/presentation/pages/ready_for_session_page.dart

```diff
@@ -31,13 +32,20 @@ class ReadyForSessionPage extends StatelessWidget {
         text = "So much energy 🔋⚡🔋⚡🔋⚡🔋\nLet's begin";
     }
 
-    return PageEnclosureMolecule(
+     return PageEnclosureMolecule(
       scaffold: false,
       title: 'start_session',
       child: ReadyForSessionOrganism(
-        callback,
+        () => handleCallback(),
```

Comment:
You can write this code cleaner by changing  `() => handleCallback(),`  to just `handleCallback`